src/blockdev.rs

```diff
@@ -202,6 +202,15 @@ pub fn reread_partition_table(file: &mut File) -> Result<()> {
                         return Err(err).chain_err(|| "couldn't reread partition table");
                     }
                 } else {
+                    let status = std::process::Command::new("kpartx")
```

Comment:
This should run after all retries, not after every retry.

Reply:
Done